modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -103,9 +103,13 @@ data:
   spec.timeout: 10m
   spec.param.networkAttachmentDefinitionName: <network_name> <1>
   spec.param.trafficGeneratorRuntimeClassName: <runtimeclass_name> <2>
+  spec.param.trafficGeneratorImage: "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:v0.1.0" <3>
+  spec.param.vmContainerDiskImage: "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:v0.1.0" <4>
```

Comment:
These tags are not final and could change in case we find more bugs.
I suggest to move forward with them, and perform the update in a follow-up PR.